### PR TITLE
Addresses issue #2430

### PR DESCRIPTION
**Pull request by pmonks**
Comments / suggestions / edits welcome.

### CODE REVIEW

File: src/LGPL-2.0-only.xml [hunk 1]
```diff
@@ -157,6 +157,7 @@
       ordinary General Public License rather than by this special one.
     </p>
     <p>
+      GNU LIBRARY GENERAL PUBLIC LICENSE
```
Comment (xsuchy) [inline src/LGPL-2.0-only.xml:160]: Can you wrap it to <p></p>?  The same in 5 other files.

**Comment by xsuchy** [comment]
Additionally, it would be great if you could describe the commit text more descriptively. E.g.

```
Add a missing title to LGPL licenses

Fixes: #2430 
```

File: src/LGPL-2.0-only.xml (hunk 1 above)
Comment (xsuchy) [reply src/LGPL-2.0-only.xml:160]: Not sure what you mean.

Before your change, there was:
```
<p>
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```
That is one paragraph.

After your change, there is:
```
<p>
   GNU LIBRARY GENERAL PUBLIC LICENSE
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```
That is still one paragraph. And https://spdx.org/licenses/preview/LGPL-2.0+ will render it as one paragraph:
```
GNU LIBRARY GENERAL PUBLIC LICENSE TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION 
AND MODIFICATION
```

I am asking for change:
```
<p>
   GNU LIBRARY GENERAL PUBLIC LICENSE
</p>
</p>
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```

or 

```
<p>
   GNU LIBRARY GENERAL PUBLIC LICENSE
   <br/>TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 </p>
```

**Comment by jlovejoy** [comment]
Thanks @pmonks ! 

To add a bit of explanation to @xsuchy comment above about the commit text: if you put "Fixes #[issue number] in the first comment for the PR, then when that PR is merged, it will automatically close the associated issue, which is quite handy. I didn't know that before, so thought I'd pass that along in case you didn't either! (and if you did, then ignore me)

I think the tests are failing b/c the changes also need to be made to the test text files - the tests run to make sure the xml and txt files match. I'll push a commit to illustrate what I mean.

**Comment by jlovejoy** [comment]
@pmonks - I just realized that you also updated the deprecated LGPLs - I don't think we need to?? (sorry, should have caught this in the issue)

**Comment by pmonks** [comment]
@jlovejoy if they're not updated, they'll never match texts out in the wild. Is that the desired behaviour? If so, might it be better to remove those templates entirely instead?

It seems to me that having "broken" templates is worse than not having them; if they exist there's an expectation that they're correct.

**Comment by jlovejoy** [comment]
@pmonks - ah, okay. wasn't sure if that was an issue or how that works. Yes, then we should update them too!

**Comment by pmonks** [comment]
@jlovejoy yeah I'm not using the matching templates for deprecated licenses myself, but I have no way of knowing if others might be, so updated them out of an abundance of caution.

File: src/LGPL-2.0-only.xml (hunk 1 above)
Comment (jlovejoy) [reply src/LGPL-2.0-only.xml:160]: FYI - appreciate the attention to detail @xsuchy and I think my commits may have perpetuated the lack of line break (personally, I prefer the <br/> for this case :) - but if the tests pass, I'm going to merge this. we can see how it looks and do any formatting updates later, if need be.

**Pull request merged**